nncf/quantization/quantization_metric.py

```diff
@@ -0,0 +1,273 @@
+import numpy as np
```

Comment:
Since this file is already in the `quantization` scope, no need to keep `quantization` in the filename.